app/api/v1/models/election.py

```diff
@@ -46,6 +51,38 @@ class CiphertextElectionContext(Base):
     crypto_extended_base_hash: str
     """The `extended base hash code (𝑄')` in [ElectionGuard Spec](https://github.com/microsoft/electionguard/wiki)"""
 
+    @staticmethod
+    def stringToElementModP(value: str) -> ElementModP:
+        elementModP = hex_to_p(value)
+        if elementModP is None:
+            raise HTTPException(
+                status_code=status.HTTP_400_BAD_REQUEST, detail="invalid key"
```

Comment:
This exception should be thrown at a high level. This should be a type exception that is caught at a higher level. The HTTPException shouldn't be mixed within the mapping level.

https://docs.python.org/3/library/exceptions.html#TypeError

Reply:
Also the detail is different between line 59 and 68.